source/changelog.txt

```diff
@@ -192,6 +192,23 @@ v2.1.3
 
 `Full release notes available on JIRA <https://jira.mongodb.org/secure/ReleaseNote.jspa?projectId=17381&version=37823>`__.
 
+v2.1.1
+------
+
+*Released December 5, 2023*
+
+- :issue:`MONGOSH-1628` - Statically linking kerberos leads to OpenSSL 
+  version conflict on RHEL8 distros 
+- :issue:`MONGOSH-1651` - Sample sessions sent to telemetry
+- :issue:`MONGOSH-1652` - Disable initial showCommands from telemetry
+- :issue:`MONGOSH-1653` - Add a new "Startup Time" event for sessions
+- :issue:`MONGOSH-1654` - Re-enable arm64 executable connectivity tests
+- :issue:`MONGOSH-1656` - Allow startup timings to be grouped
+- :issue:`MONGOSH-1658` - node-worker-runtime: remove function 
+  properties from errors before serialization
+
+`Full release notes available on JIRA <https://jira.mongodb.org/secure/ConfigureReleaseNote.jspa?projectId=17381&version=37380>`__.
```

Comment:
[question] Should this link lead to the JIRA tickets as opposed to the page to configure the release notes?